mappers/Baseline.rb

```diff
@@ -270,13 +270,22 @@ def time_mapping(time)
               end
             end
 
-          begin
-            weather_filename = feature.weather_filename
-            if !feature.weather_filename.empty?
-              OpenStudio::Extension.set_measure_argument(osw, 'ChangeBuildingLocation', 'weather_file_name', weather_filename)
-            end
-          rescue
-          end 
+            # set weather file
+            begin
+              weather_filename = feature.weather_filename
+              if !feature.weather_filename.empty?
+                OpenStudio::Extension.set_measure_argument(osw, 'ChangeBuildingLocation', 'weather_file_name', weather_filename)
+                puts "Setting weather_file_name to #{weather_filename} as specified in the FeatureFile"
+              else
+                epw_file_path = Dir.glob(File.join(File.dirname(__FILE__), '../weather/*.epw'))[0]
```

Comment:
@kflemin currently, the code doesn't go to the else statement if the feature.weather_filename is empty.